glossary/kontrol-toml.md

```diff
@@ -0,0 +1,306 @@
+# Configuration Options for Kontrol TOML file
+
+The following is a kontrol.toml file with all configuration options set.
+
+```toml
+[build.default]
+foundry-project-root       = '.'
+regen                      = false
+rekompile                  = false
+verbose                    = false
+debug                      = false
+require                    = 'lemmas.k'
+module-import              = 'TestBase:KONTROL-LEMMAS'
+auxiliary-lemmas           = true
+
+# ? definition_dir = None
+# ? spec_module = None
+
+[build]
+# Directories to lookup K definitions in.
```

Comment:
I wouldn't put comments below or after the options. How about placing them next to the option instead? E.g., 
 ```
# include = [] # Directories to lookup K definitions in.
```